PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -1254,13 +1256,49 @@ struct HfTrackIndexSkimCreator {
     }
   }
 
+  /// Method to perform selections on difference from nominal mass for phi decay
+  /// \param pVecTrack0 is the momentum array of the first daughter track
+  /// \param pVecTrack1 is the momentum array of the second daughter track
+  /// \param pVecTrack2 is the momentum array of the third daughter track
+  /// \param cutStatus is a 2D array with outcome of each selection (filled only in debug mode)
+  /// \param whichHypo information of the mass hypoteses that were selected
+  /// \param isSelected is a bitmap with selection outcome
+  /// \param iDecay3P is the index of the 3-prong decay
+  template <typename T1, typename T2, typename T3>
+  void isPhiDecayPreselected(T1 const& pVecTrack0, T1 const& pVecTrack1, T1 const& pVecTrack2, T2& cutStatus, T3& whichHypo, int& isSelected, int const& iDecay3P)
```

Comment:
There is no need of `iDecay3P` since this applies only to the Ds
```suggestion
  template <typename T1, typename T2, typename T3>
  void isPhiDecayPreselected(T1 const& pVecTrack0, T1 const& pVecTrack1, T1 const& pVecTrack2, T2& cutStatus, T3& whichHypo, int& isSelected)
```